app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2 class="fs-4 p-3 border border-1 container mt-5 p-2 bio">Log in</h2>
+<div class="fs-4 p-3 border border-1 container mt-5 p-2 bio">
+  <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email", class: "form-control mb-4" %>
+  </div>
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class: "form-control mb-4" %>
+  </div>
+  <% if devise_mapping.rememberable? %>
+  <div class="field">
+    <%= f.check_box :remember_me %>
+    <%= f.label :remember_me %>
+  </div>
+  <% end %>
+  <div class="actions">
+    <%= f.submit "Log in", data: {disable_with: "Login..."}, class: "btn btn-primary my-3"%>
+  </div>
+  <% end %>
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] Also, I am not able to sign-up or sign-in, it says `you have to confirm your email address before continuing` but notification never comes and because of this I am unable to go to the other pages after `sign-up` or `sign-in`.

![Screenshot from 2022-01-06 02-05-32](https://user-images.githubusercontent.com/22434660/148285598-12c26c79-5861-42f0-99da-3eeff9420265.png)

Reply:
Hi @Dipeshtwis , 
Thank you for reviewing my works and your comments. I made the changes you requested.

Cheers!